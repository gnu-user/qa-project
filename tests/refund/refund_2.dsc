TITLE
    Refund Buyer Input (Invalid Input, buyer does not exist)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the refund 
    transaction. When prompted, enter a username which does not exist. Verify 
    that the buyer is not accepted, and an error occurs.
DEPENDENCIES
    refund_1
