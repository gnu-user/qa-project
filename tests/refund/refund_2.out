Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the buyer's username: 
Invalid username, username does not exist.
> 